Earthfile

```diff
@@ -280,6 +280,7 @@ examples:
     BUILD ./examples/ruby-on-rails+docker
     BUILD ./examples/scala+docker
     BUILD ./examples/cobol+docker
+    BUILD ./examples/rust
```

Comment:
A target is needed here, like `BUILD ./examples/rust+docker`

Reply:
Oh doh!

Reply:
Thanks